### PR TITLE
Update the release date of v1.3.9

### PR DESCRIPTION
**Pull request by ahrtr**
Link to https://github.com/etcd-io/bbolt/issues/696

### CODE REVIEW

**Comment by ahrtr** [comment]
cc @Elbehery  @fuweid @jmhbnz @serathius

File: CHANGELOG/CHANGELOG-1.3.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Note that we start to track changes starting from v1.3.7.
 
-## v1.3.9(TBD)
+<hr>
+
+## v1.3.9(2024-02-24)
```
Comment (jmhbnz) [inline CHANGELOG/CHANGELOG-1.3.md:5]: Should we add a new TBD entry for `1.3.10`?  Or will we not have any more patch releases for this branch?

**Review by jmhbnz — APPROVED** [review]
LGTM - Just a question about future patch releases below.

File: CHANGELOG/CHANGELOG-1.3.md (hunk 1 above)
Comment (ahrtr) [reply CHANGELOG/CHANGELOG-1.3.md:5]: We can add that when we have any new fix for 1.3.10. Most likely we won't have any fix for 1.3.10 in near future.

**Review by fuweid — APPROVED** [review]
LGTM

**Pull request merged**